### ISSUE TITLE
[Publications] Create a Demo Page

### ISSUE THREAD

**Issue by sabrinang** [mozilla/foundation.mozilla.org#5827]
We can test out https://github.com/mozilla/foundation.mozilla.org/issues/5765 how-to docs and with the result it can be a demo page for [How do I Wagtail](https://foundation.mozilla.org/en/docs/how-do-i-wagtail/) so users can see what is possible with publication page types.

- Review How To doc
- Input content _(example: check out https://www.gutenberg.org/ for a public domain story as dummy text to input?)_
- Collect feedback/QA for publication templates and How To doc itself
- Publish demo page

**Comment by nancyt1** [comment]
Made the [demo page](https://foundation.mozilla.org/en/docs/how-do-i-wagtail/grimms-fairy-tales-demo/) and set it to private so it can be viewed if you're logged in. Also wrote my feedback in this [google doc](https://docs.google.com/document/d/136l3IIV2HTdG3cJMxtAZLgRRA4lEdrpoe5mUbdIhT_Q/edit). Probably ready for someone to review (:

**Comment by sabrinang** [comment]
note: assigned myself to review

**Comment by sabrinang** [comment]
Both of these are so great, thank for the thorough job!

I reviewed the feedback doc and distributed the feedback to [feature requests epic](https://github.com/mozilla/foundation.mozilla.org/issues/6119) and [ux issue](https://github.com/mozilla/foundation.mozilla.org/issues/5800) and will keep referencing the doc. Great feedback and insights!

I reviewed https://foundation.mozilla.org/en/docs/how-do-i-wagtail/grimms-fairy-tales-demo/ and have a few pieces of feedback:
- for the title we should probably include "publication" with demo or sample page somewhere as there is another "sample page" on https://foundation.mozilla.org/en/docs/how-do-i-wagtail/publications/ so that's clear its a publication demo
- on https://foundation.mozilla.org/en/docs/how-do-i-wagtail/grimms-fairy-tales-demo/collection-of-fairy-tales/haensel-and-grethal/ & https://foundation.mozilla.org/en/docs/how-do-i-wagtail/grimms-fairy-tales-demo/wolf-and-seven-little-kids/, when h2 is used it makes a summary box which looks a bit funny when it is only one — should h3 be used instead? Do you think users would want to use h2 otherwise without a summary box on shorter length content though?
- we may want to include an author example, "table" and "2x image" component somewhere to see how it looks like.

It's fun too see all this content testing out the templates, other than those few pieces of feedback to be updated it'd be good to publish!

**Comment by nancyt1** [comment]
1. Updated the title to be "Publication Demo — Grimm's Fairy Tales"
2. I changed it to h3, mostly used h2 as an example of summary box but I've already done that somewhere else. I feel like h3 would be fine for most cases but I can see users potentially wanting to use h2 without the summary box.
3. Added examples of authors, a table in "Foreward", and 2x Image in "Haensel and Grethel" pages. 

Also glad to be of help!
